<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formSlideSettings</class>
 <widget class="QDialog" name="formSlideSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slide settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lstImgs.qrc">
    <normaloff>:/new/icons/imagenInte/document-properties.svg</normaloff>:/new/icons/imagenInte/document-properties.svg</iconset>
  </property>
  <widget class="QSpinBox" name="spinBoxIni">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>16</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Jump degree</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial degree</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpeed">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>136</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>800</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>196</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="lstImgs.qrc">
     <normaloff>:/new/icons/imagenInte/document-save.svg</normaloff>:/new/icons/imagenInte/document-save.svg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>196</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="icon">
    <iconset resource="lstImgs.qrc">
     <normaloff>:/new/icons/imagenInte/gtk-cancel.svg</normaloff>:/new/icons/imagenInte/gtk-cancel.svg</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEnd">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>56</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxJump">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>96</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Final degree</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="lstImgs.qrc"/>
 </resources>
 <connections/>
</ui>
